core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -6050,6 +6050,14 @@ public String toString() {
               + "when Alluxio workers are required but not ready.")
           .setScope(Scope.CLIENT)
           .build();
+  public static final PropertyKey USER_HDFS_LISTSTATUS_MOUNTINFO_DISABLE =
```

Comment:
let's rename this property to USER_HDFS_CLIENT_EXCLUDE_MOUNT_INFO_ON_LIST_STATUS